docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+# Automated Foundational Architecture for PCI DSS-Compliant Environments (v1)
+
+### PUBLIC PREVIEW BETA 1
+
+> [EDIT: INSERT "AZURE CYBER BLUEPRINT" PROGRAM OVERVIEW]
+
+## Overview
+
+The Automated Foundational Architecture for PCI DSS-compliant Environments (v1) provides guidance for the deployment of a PCI DSS-compliant Platform-as-a-Service (PaaS) environment suitable for handling sensitive payment card data. It showcases a common reference architecture and is designed to simplify adoption of Microsoft Azure. This foundational architecture (developed jointly with [Avyan Consulting](http://www.avyanconsulting.com/azureservices), a Microsoft MVP partner) illustrates an end-to-end solution to meet the needs of organizations seeking a cloud-based approach to reducing the burden and cost of deployment.
```

Comment:
remove Avyan from the intro - OK to credit at the end of the doc (compare SharePoint and SAP RA's)